api/popUpInfo/alsB.json

```diff
@@ -0,0 +1,16 @@
+  
+{
+    "restaurant": "Al’s Breakfast",
+    "location": {
+        "latitude": 44.981062, 
+        "longitude": -93.235666 
+    },
+    "info": 
+        {
+            "desc": "Small breakfast place: egg specials, pancakes, waffles",
+            "rating": "Star Rating: 5",
```

Comment:
Ratings will change depending on user input. Please don't put dynamic data in static JSON files.